modules/apps/commerce/commerce-currency-web/src/main/java/com/liferay/commerce/currency/web/internal/display/context/CommerceCurrenciesDisplayContext.java

```diff
@@ -236,25 +239,21 @@ else if (navigation.equals("inactive")) {
 				getOrderByCol(), getOrderByType()));
 		_searchContainer.setOrderByType(getOrderByType());
 
-		if (active != null) {
-			boolean navigationActive = active;
+		_searchContainer.setResultsAndTotal(
+			_commerceCurrencyService.searchCommerceCurrencies(
+				themeDisplay.getCompanyId(), getKeywords(), active,
+				_searchContainer.getStart(), _searchContainer.getEnd(),
+				CommerceCurrencyUtil.getCommerceCurrencySort(
+					getOrderByCol(), getOrderByType())));
 
+		if (Validator.isBlank(getKeywords())) {
```

Comment:
I think you don't need this just manage the case keywords are empry while making the searchContext, anyway this should be put before the oder setResultAndTotal L242 and the other should be in an else block

Reply:
@andrea-ale-sbarra I have added the else block as per your advice however to manage the empty case keywords in the serachContext is not something that I can get it to work